### PR TITLE
add lint & breaking change to ci-fix.md

### PR DESCRIPTION
**Pull request by jianyexi**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: documentation/ci-fix.md [hunk 1]
```diff
@@ -44,3 +44,28 @@ oav validate-spec <swagger-spec-path>
...
+
+## Breaking Change Check
+
+run oad locally (the breaking change is reported by oad tool):
+```
+npm install -g oad
+oad compare <old-spec-path> <new-spec-path> 
+```
+Please see [readme](https://github.com/Azure/openapi-diff/blob/master/README.md) for how to install or run tool in details.
+Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/diff).
```
Comment (akning-ms) [inline documentation/ci-fix.md]: Pls remove this link to openapi hub. it is not for breaking change

Comment (jianyexi) [reply documentation/ci-fix.md:56]: actually they all use oad to diff, I don't find anything different

Comment (akning-ms) [reply documentation/ci-fix.md:56]: In openapi hub, it is just a compare tool. not for breaking change you can try it, then you can understand why

Comment (jianyexi) [reply documentation/ci-fix.md:56]: I have try it , the compare tool  is also oad,  the breaking change is based on result of oad

Comment (akning-ms) [reply documentation/ci-fix.md:56]: Ok, got your mean, thanks Jianye

**Review by josefree — APPROVED** [review]

**Pull request merged**